plugins/meta/portmap/chain.go

```diff
@@ -70,7 +70,10 @@ func (c *chain) teardown(ipt *iptables.IPTables) error {
 	// flush the chain
 	// This will succeed *and create the chain* if it does not exist.
 	// If the chain doesn't exist, the next checks will fail.
-	if err := ipt.ClearChain(c.table, c.name); err != nil {
+	err := ipt.ClearChain(c.table, c.name)
+	eerr, eok := err.(*iptables.Error)
```

Comment:
what if eok is false?

Reply:
the if below checks if err!=nil, can eok be false if err=nil?
maybe should I change the if and use eok instead or I'm misunderstanding how eok and err works?

Reply:
I mean `err` maybe a normal error instead of an iptables Error, this may happen in [https://github.com/coreos/go-iptables/blob/master/iptables/iptables.go#L453](https://github.com/coreos/go-iptables/blob/master/iptables/iptables.go#L453) .

Reply:
got it, thanks